tests/components/StartErrorModal.test.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+import StartErrorModal from '../../app/js/components/manageApps/StartErrorModal.jsx';
+
+describe('<StartErrorModal />', () => {
+  const app =
+  {
+    "appDetails":
+    {
+      "version": "0.1.0",
+      "name": "openmrs-addOnManager",
+      "description": "This will help in uploading the openmrs open web apps and omod files",
+      "icons":
+      {
+        "16": null,
+        "48": "/img/omrs-button.png",
+        "128": null,
+      },
+      "developer":
+      {
+        "url": "https://github.com/openmrs/openmrs-owa-addonmanager",
+        "name": "andela-pupendo,kingisaac95, andela-wanjikum, andela-jomadoye",
```

Comment:
@dkayiwa recommended that we use the name `Andela Team` here to reduce the clutter of names from different Andela developers who will/ have worked on the modules

Reply:
@LWanjiru This is mock data for a test